isar/lib/src/sentry_isar.dart

```diff
@@ -0,0 +1,253 @@
+import 'package:isar/isar.dart';
+import 'package:meta/meta.dart';
+import 'package:sentry/sentry.dart';
+import 'version.dart';
+
+import 'sentry_isar_collection.dart';
+import 'sentry_span_helper.dart';
+
+/// A sentry wrapper around the Isar Database
+@experimental
+class SentryIsar implements Isar {
+  @internal
+  // ignore: public_member_api_docs
+  static const dbOp = 'db';
+
+  @internal
+  // ignore: public_member_api_docs
+  static const dbSystemKey = 'db.system';
+  @internal
+  // ignore: public_member_api_docs
+  static const dbSystem = 'isar';
+
+  @internal
+  // ignore: public_member_api_docs
+  static const dbNameKey = 'db.name';
+
+  @internal
+  // ignore: public_member_api_docs
+  static const dbCollectionKey = 'db.collection';
+
+  final Isar _isar;
+  final Hub _hub;
+  final _spanHelper = SentrySpanHelper(
+    // ignore: invalid_use_of_internal_member
+    SentryTraceOrigins.autoDbIsar,
+  );
+
+  /// ctor of SentryIsar
+  SentryIsar(this._isar, this._hub) {
+    _spanHelper.setHub(_hub);
```

Comment:
Please add a version.dart and add `integrations` and `package` to the hub

See: [here](https://github.com/getsentry/sentry-dart/blob/main/sqflite/lib/src/version.dart) and [here](https://github.com/getsentry/sentry-dart/blob/041d84de0540b1da829433f102c870bac73aec79/sqflite/lib/src/sentry_database.dart#L68)

Reply:
This is already part of the PR, or am I missing something else?

Version
https://github.com/getsentry/sentry-dart/blob/fe19113cfa38ba221bd27283d0a6990fce1c21ed/isar/lib/src/version.dart#L2

Integration
https://github.com/getsentry/sentry-dart/blob/fe19113cfa38ba221bd27283d0a6990fce1c21ed/isar/lib/src/sentry_isar.dart#L44